sources/org.osbuild.files

```diff
@@ -110,7 +110,7 @@ def main(options, checksums, cache, output):
             try:
                 if isinstance(urls[checksum], dict):
                     # check if url needs rhsm secrets
-                    if urls[checksum].get("secrets").get("name") == "org.osbuild.rhsm":
+                    if urls[checksum].get("secrets", {})["name"] == "org.osbuild.rhsm":
```

Comment:
Sorry, I think I confused you. I think you do indeed want `urls[checksum].get("secrets", {}).get("name")` to make this fully optional.

Reply:
Haha you right.